@kiva/kv-components/src/vue/KvLendCta.vue

```diff
@@ -93,217 +99,181 @@
+					</div>
+				</div>
 
-					<!-- Stranded loans (mobile) -->
-					<kv-lend-amount-button
-						v-if="isLendAmountButton && !enableFiveDollarsNotes"
-						class="tw-w-full"
-						:loan-id="loanId"
-						:show-now="false"
-						:amount-left="amountForLendAmountButton"
-						:is-adding="isAdding"
-						@add-to-basket="addToBasket"
-					/>
-				</fieldset>
-			</div>
-			<!-- ===== END MOBILE-ONLY ===== -->
-
-			<!-- ===== DESKTOP/TABLET (original) ===== -->
-			<div class="tw-hidden md:tw-block tw-w-full">
-				<fieldset
-					class="tw-w-full tw-flex"
+				<!-- Desktop-only preset buttons (hidden on mobile when showing special content) -->
+				<div
+					v-if="showPresetAmounts && !isAdding"
 					:class="{
-						'tw-flex-col md:tw-flex-row md:tw-justify-between tw-min-w-0': showPresetAmounts,
-						'tw-gap-1.5': showPresetAmounts && !isLendAmountButton
+						'tw-flex tw-gap-1 lg:tw-gap-2 tw-flex-wrap md:tw-flex-nowrap': true,
+						'tw-hidden md:tw-flex': !isLendAmountButton && presetButtonsPrices.length > 1
 					}"
-					:disabled="isAdding"
-					data-testid="bp-lend-cta-select-and-button"
 				>
-					<div
-						v-if="showPresetAmounts && !isAdding"
-						class="tw-flex tw-gap-1 lg:tw-gap-2 tw-flex-wrap md:tw-flex-nowrap"
-					>
-						<template v-if="!isLendAmountButton">
-							<kv-ui-button
-								v-for="option in presetButtonsPrices"
-								:key="option"
-								variant="secondary"
-								class="tw-inline-flex tw-flex-1 preset-option tw-w-8"
-								:class="{'selected-option': selectedOption == option }"
-								data-testid="bp-lend-cta-lend-button"
-								@click="clickPresetButton(option)"
-							>
-								${{ option }}
-							</kv-ui-button>
-						</template>
-						<kv-ui-select
-							v-if="showFilteredDropdown"
-							:id="`LoanAmountDropdown_${loanId}`"
-							v-model="selectedDropdownOption"
-							class="tw-min-w-12 tw-rounded filtered-dropdown tw-w-full"
-							:class="{
-								'unselected-dropdown': !selectedDropdown,
-								'selected-dropdown': selectedDropdown,
-							}"
-							aria-label="Lend amount"
-							@update:modelValue="trackLendAmountSelection"
-							@click.native.stop="clickDropdown"
-						>
-							<option
-								v-for="priceOption in presetDropdownPrices"
-								:key="priceOption"
-								:value="priceOption"
-							>
-								{{ priceOption !== OTHER_OPTION ? `$${priceOption}` : priceOption }}
-							</option>
-						</kv-ui-select>
-					</div>
-
-					<div
-						v-else-if="!showPresetAmounts"
-						class="amountDropdownWrapper"
-					>
-						<kv-ui-select
-							v-if="hideShowLendDropdown && !isLessThan25"
-							:id="`LoanAmountDropdown_${loanId}`"
-							v-model="selectedOption"
-							class="tw-min-w-12"
-							style="border-radius: 14px 0 0 14px;"
-							aria-label="Lend amount"
-							@update:modelValue="trackLendAmountSelection"
-							@click.native.stop="clickDropdown"
+					<template v-if="!isLendAmountButton">
+						<kv-ui-button
```

Comment:
I'm noticing that comparing to the previous implementation, the desktop buttons have more padding between them. Please match the old padding, which can be seen on the deployed storybook linked below.

https://main--608b4cf87f686c00213841b1.chromatic.com/?path=/story/kvlendcta--with-preset-options-loading

<img width="1561" height="313" alt="Image" src="https://github.com/user-attachments/assets/d0efd62e-19db-4f5e-8eec-dc87ef6d9d12" />